Extension/package.json

```diff
@@ -379,7 +379,10 @@
           "enum": [
             "msvc-x64",
             "gcc-x64",
-            "clang-x64"
+            "clang-x64",
```

Comment:
thanks for catching this.

Reply:
Yeah, I noticed it by accident when it came up in a search.